src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,38 @@
-export const TodoInfo = () => {};
+import classNames from 'classnames';
+import './TodoInfo.scss';
+import { UserInfo } from '../UserInfo';
+
+import { Todo } from '../../types';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const {
+    id,
+    title,
+    completed,
+    user,
+  } = todo;
+
+  return (
+    <div className="todo-content">
+      <article
+        data-id={id}
+        className={classNames(
+          'TodoInfo',
+          { 'TodoInfo--completed': completed },
+        )}
+      >
+
+        <h2 className="TodoInfo__title">
```

Comment:
Consider using a more descriptive class name for the title element, like `TodoInfo__title-text`. 📝